jenkins_jobs/trigger_host_os_build_from_builds_repo.groovy

```diff
@@ -23,6 +23,7 @@ job('trigger_host_os_build_from_builds_repo') {
       extensions {
 	commitStatus {
 	  context('Build Host OS')
+	  statusUrl('${JENKINS_URL}/job/build_host_os/${TRIGGERED_BUILD_NUMBER_build_host_os}')
```

Comment:
OK. (1) Not knowing if the variable is a Jenkins job variable, a global Jenkins custom variable or a global Jenkins implicit variable and (2) having no doc about the meaning of these variables in the script (in this case, groovy script) which uses them really annoy me, but never mind.